### DISCUSSION TITLE
Time for 0.9.6?

### DISCUSSION THREAD

**Discussion by liu-song-6** [dynup/kpatch#1259] [Q&A] [answered]
Hi folks,

We would like to ship the latest create-diff-object fix (the segfault). Since it has been a while since 0.9.5 release, it is ideal to ship the fixes with 0.9.6. Shall we cut a new release now?

Thanks!

**Comment by joe-lawrence** [comment, marked as answer]
Hi @liu-song-6 : yeah, it's been a while since the last release.  I'll put in on my TODO for tomorrow, thanks for reminding.

**Reply by joe-lawrence** [reply]
Done: https://github.com/dynup/kpatch/releases/tag/v0.9.6

**Reply by liu-song-6** [reply]
Thanks!
